td4m3/src/main/java/ca/td/td4m3/components/mortgage/binder/MortgageBinder.kt

```diff
@@ -0,0 +1,18 @@
+package ca.td.td4m3.components.mortgage.binder
+
+import android.view.ViewGroup
+import ca.td.td4m3.R
+import ca.td.td4m3.components.mortgage.viewholder.MortgageViewHolder
+import com.ngam.rvabstractions.AbstractDataBinder
+
+class MortgageBinder: AbstractDataBinder<MortgageViewHolder>() {
+
+    override fun createViewHolder(parent: ViewGroup): MortgageViewHolder {
+        return MortgageViewHolder(getView(MortgageViewHolder.getLayoutId(), parent))
+    }
+
+    override fun bindViewHolder(viewHolder: MortgageViewHolder) {
+        viewHolder.imageView.setBackgroundResource(R.drawable.mortgage_picture)
```

Comment:
Why do we need to do this through this?
Why not set image directly in td_mortgage.xml?

If the idea is you want to be able to change the image, I recommend having a constructor take in a drawableResId and setting it that way.

Reply:
Yah will simply put in xml file next pr.